frontends/infinite-corridor/src/api/fields/factories.ts

```diff
@@ -1,5 +1,5 @@
 import { faker } from "@faker-js/faker/locale/en"
-import { makePaginatedFactory, Factory } from "ol-util/src/factories"
+import { makePaginatedFactory, Factory } from "ol-util/factories"
```

Comment:
When I added filemock.js and svgmock.js to `ol-util/test-utils`, I changed `ol-util/package.json` to use the "exports" field so we don't need /src.